docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '2'
+
+services:
+  litecoind:
+    build: ./data-sources/litecoind
+    volumes:
+      - ./data-sources/litecoind/litecoin.conf:/home/litecoin/.litecoin/litecoin.conf
```

Comment:
You can define a volume as a single file that is shared between the host and the docker container.